ee/vellum_ee/workflows/display/nodes/vellum/base_node.py

```diff
@@ -0,0 +1,21 @@
+from uuid import UUID
+from typing import Any, Generic, Optional, TypeVar
+
+from vellum.workflows.nodes.bases.base import BaseNode
+from vellum.workflows.types.core import JsonObject
+from vellum_ee.workflows.display.nodes.base_node_vellum_display import BaseNodeVellumDisplay
+from vellum_ee.workflows.display.types import WorkflowDisplayContext
+
+_BaseNodeType = TypeVar("_BaseNodeType", bound=BaseNode)
+
+
+class BaseNodeDisplay(BaseNodeVellumDisplay[_BaseNodeType], Generic[_BaseNodeType]):
+    def serialize(
+        self, display_context: WorkflowDisplayContext, error_output_id: Optional[UUID] = None, **kwargs: Any
```

Comment:
Generic nodes won't have the `Reject on Error` toggle, hence we shouldn't allow it as a kwarg

```suggestion
        self, display_context: WorkflowDisplayContext, **kwargs: Any
```